src/almanack/cli.py

```diff
@@ -138,7 +138,7 @@ def check(
 
         # prepare almanack score output
         almanack_score_output = (
-            f"Software Gardening Almanack score: {100 * almanack_score_metrics['almanack-score']:.2f}% "
+            f"Software Gardening Almanack check ratio: {100 * almanack_score_metrics['almanack-score']:.2f}% "
```

Comment:
```suggestion
            f"Software Gardening Almanack summary: {100 * almanack_score_metrics['almanack-score']:.2f}% passed or failed "
```
Maybe instead of a percentage, make it a score out of the number of metrics. It might feel better to someone to know they at least did well in 3/8 aspects or passed 3/8 "standards". 

It might be good to also output the things the repo has, along with what it doesn't have. For example, I got a score of 45.45%, but what did my repo have that got me that score?  I only know what my repo didn't have.

Reply:
Thanks - I like the idea of "summary" here. I don't think we can accurately use "passed or failed" in the sentence  because the ratio shows "passing out of total". That said, I like the idea of including additional "verbose" options here to show the passing checks as well because this would provide a better context for the user. I think this should be work for a different PR because it'd require reengineering how the checks data are returned.